File: trail1/1부터 곱셈하기/multiple-from-one.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int prod = 1, mark = 1;
        for (int i=1; i<=10; i++) {
            prod *= i;
            if (prod >= N) {
                mark = i;
                break;
            }
        }
        Console.WriteLine(mark);
    }
}

File: trail1/1부터 곱셈하기/multiple-from-one.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    int prod = 1;
    int mark = 0;
    for (int i=1; i<=10; i++) {
        prod *= i;
        if (prod >= N) {
            mark = i;
            break;
        }
    }
    cout << mark << " ";
    return 0;
}